tests/unit/test_viz_sankey.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+from unittest.mock import patch
+import pandas as pd
+from dune_client.viz.graphs import create_sankey
+
+
+class TestCreateSankey(unittest.TestCase):
+
+    # Setting up a dataframe for testing
+    def setUp(self):
+        self.df = pd.DataFrame(
```

Comment:
Could this test be made much smaller and still demonstrate the point? Like with two tokens?

Reply:
good point shortened!